src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerFactoryExtensions.cs

```diff
@@ -42,5 +50,115 @@ public static ILoggingBuilder AddConsole(this ILoggingBuilder builder, Action<Co
 
             return builder;
         }
+
+        /// <summary>
+        /// Add and configure a console log formatter named 'json' to the factory.
+        /// </summary>
+        /// <param name="builder">The <see cref="ILoggingBuilder"/> to use.</param>
+        /// <param name="configure">A delegate to configure the <see cref="ConsoleLogger"/> options for the built-in default log formatter.</param>
+        public static ILoggingBuilder AddSimpleConsole(this ILoggingBuilder builder, Action<SimpleConsoleFormatterOptions> configure)
```

Comment:
maybe you have internal method: 

internal static ILoggingBuilder AddConsoleWithFormatter(ILoggingBuilder builder, Action<SimpleConsoleFormatterOptions> configure, 
ConsoleFormatterNames name)
        {
            if (configure == null)
            {
                throw new ArgumentNullException(nameof(configure));
            }

            builder.AddConsole((ConsoleLoggerOptions options) => options.FormatterName = name);
            builder.Services.Configure(configure);

            return builder;
        }


then you can write the methods: 


public static ILoggingBuilder AddSimpleConsole(this ILoggingBuilder builder, Action<SimpleConsoleFormatterOptions> configure) =>
AddConsoleWithFormatter(builder, configure, ConsoleFormatterNames.Simple);

public static ILoggingBuilder AddJsonConsole(this ILoggingBuilder builder, Action<SimpleConsoleFormatterOptions> configure) =>
AddConsoleWithFormatter(builder, configure, ConsoleFormatterNames.Json);

public static ILoggingBuilder AddSystemdConsole(this ILoggingBuilder builder, Action<SimpleConsoleFormatterOptions> configure) =>
AddConsoleWithFormatter(builder, configure, ConsoleFormatterNames.Systemd);